requirements.txt

```diff
@@ -17,13 +17,13 @@ Pygments==2.2.0
 pyparsing==2.2.2
 python-slugify==1.2.6
 pytz==2018.5
-PyYAML==3.13
+PyYAML==6.0
```

Comment:
I had hard time to get the doc working on MacOS:

I've done: 
source venv/bin/activate
python --version
Python 3.9.1

So I don't know if it works on other os

Reply:
What is the minimum python requirements ? Is PyYAML 6.0 compatible with that python version ?

Reply:
Seems to be working with python 3.6 so OK for me.

Reply:
We "expect python 3" for this project. It's written in the README but I can't find something that check it programmatically. 

Concerning PyYAML. It is not easy to answer your question... https://pypi.org/project/PyYAML/ from there it seems it requires python >= 3.6. But when I read https://github.com/yaml/pyyaml/blob/master/setup.py#L30-L35 or https://github.com/yaml/pyyaml/blob/master/tox.ini#L2 I think that 3+ is supported but not sure how to handle that. 

Instead of having Python 3.9 for me, may 3.6 for you, is there a way to fix the python version for this (cozy-doc) project?